7/apache/Dockerfile

```diff
@@ -1,6 +1,5 @@
 # from https://www.drupal.org/docs/8/system-requirements/drupal-8-php-requirements
-FROM php:7.2-apache-stretch
-# TODO switch to buster once https://github.com/docker-library/php/issues/865 is resolved in a clean way (either in the PHP image or in PHP itself)
+FROM php:7.2-apache
```

Comment:
I think we should probably keep an explicit Debian release name (so `-buster`), just to control when it will change to avoid bugs similar to the deleted comment. I don't think `bullseye` will become Debian Stable until ~2021, but I'd still like to stay prepared.